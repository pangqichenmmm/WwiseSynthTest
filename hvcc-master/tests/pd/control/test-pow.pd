#N canvas 750 181 185 214 10;
#X obj 34 40 loadbang;
#X obj 34 156 print;
#X msg 34 92 2;
#X msg 66 92 -2;
#X obj 34 62 t b b b b;
#X msg 127 92 -5;
#X msg 96 92 3;
#X obj 34 134 pow 2;
#X connect 0 0 4 0;
#X connect 2 0 7 0;
#X connect 3 0 7 1;
#X connect 4 0 2 0;
#X connect 4 1 3 0;
#X connect 4 2 6 0;
#X connect 4 3 5 0;
#X connect 5 0 7 0;
#X connect 6 0 7 0;
#X connect 7 0 1 0;
