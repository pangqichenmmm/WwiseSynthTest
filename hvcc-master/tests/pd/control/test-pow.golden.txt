[@ 0.000] print: 0
[@ 0.000] print: 9
[@ 0.000] print: 0.25